android/build.gradle

```diff
@@ -43,7 +43,7 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        implementation 'com.hypertrack:hypertrack:6.3.0'
+        implementation 'com.hypertrack:hypertrack:6.4.0'
```

Comment:
why not moving it to variables to make it easy to modify?

Reply:
Because the goal was to do the change ASAP :)
I would postpone this to the next stage, when we will automate all the stuff including changing versions